src/main/java/org/jabref/model/pdf/FileAnnotationType.java

```diff
@@ -12,14 +22,33 @@
     FREETEXT("FreeText"),
     STRIKEOUT("Strikeout"),
     LINK("Link"),
+    UNKNOWN("Unknown"),
     NONE("None");
 
     private final String name;
+    private static final Log LOGGER = LogFactory.getLog(FileAnnotationType.class);
+
 
     FileAnnotationType(String name) {
         this.name = name;
     }
 
+    /**
+     * Determines the FileAnnotationType of a raw PDAnnotation. Returns 'UNKNOWN' if the type is currently not in our
+     * list of FileAnnotationTypes.
+     *
+     * @param annotation the raw PDAnnotation
+     * @return The determined FileAnnotationType
+     */
+    public static FileAnnotationType parseFileAnnotationType(PDAnnotation annotation) {
```

Comment:
I think we normally called such methods simply `parse` since the rest should be clear from the class name. But you can leave it also like that.